app/Models/Song.php

```diff
@@ -183,9 +185,12 @@ public function updateSingle($title, $albumName, $artistName, $lyrics, $track, $
             $album = Album::get(Artist::getVarious(), $albumName, true);
         }
 
+        $this->title = $title;
```

Comment:
Thanks. Let me fix that bug on `master` first, while waiting for the full review.